stripe/src/main/java/com/stripe/android/net/StripeApiHandler.java

```diff
@@ -191,7 +212,8 @@ static Source createSourceOnServer(
             Map<String, Object> loggingParams = LoggingUtils.getSourceCreationParams(
                     apiKey,
                     sourceParams.getType());
-            logApiCall(loggingParams, options, loggingResponseListener);
```

Comment:
It does actually get a guid and a muid added to the logging params, but that was from the original way we were doing basic mirador support. It's put into the query post parameters, rather than the header.

The reason that's done in the createSourceOnServer was because I created the Source class with its own ability to become a hashmap or a json object, so I didn't have the utility hashMapFromCard or hashMapFromBankAccount (where this is also called).

Do we no longer need those parameters?